app-with-tutorials/cap/fiori-elements-incident-management/app/incidents/README.md

```diff
@@ -0,0 +1,42 @@
+## Application Details
+|               |
+| ------------- |
+|**Generation Date and Time**<br>Fri Dec 09 2022 13:43:41 GMT+0000 (Coordinated Universal Time)|
+|**App Generator**<br>@sap/generator-fiori-elements|
+|**App Generator Version**<br>1.8.3|
+|**Generation Platform**<br>SAP Business Application Studio|
+|**Template Used**<br>List Report Page V4|
+|**Service Type**<br>Local Cap|
+|**Service URL**<br>http://localhost:4004/incident/
+|**Module Name**<br>incidents|
+|**Application Title**<br>Incidents Management|
+|**Namespace**<br>sap.fe.demo|
+|**UI5 Theme**<br>sap_horizon|
+|**UI5 Version**<br>1.108.0|
+|**Enable Code Assist Libraries**<br>False|
+|**Enable TypeScript**<br>False|
+|**Add Eslint configuration**<br>False|
+|**Main Entity**<br>Incidents|
+|**Navigation Entity**<br>None|
+
+## incidents
+
+SAP Fiori Elements
+
+### Starting the generated app
+
+-   This app has been generated using the SAP Fiori tools - App Generator, as part of the SAP Fiori tools suite.  In order to launch the generated app, simply start your CAP project and navigate to the following location in your browser:
```

Comment:
We should probably give the full extension name here so that users can search for it if required. 

```suggestion
-   This app has been generated using the SAP Fiori tools - Application Generator, as part of the SAP Fiori tools suite.  In order to launch the generated app, simply start your CAP project and navigate to the following location in your browser:
```